internal/env/env.go

```diff
@@ -0,0 +1,20 @@
+package env
+
+import (
+	"os"
+	"strconv"
+)
+
+// GetEnvBool Extracts a boolean from an env var. Falls back to the default
+// if the key is unset or not a valid boolean.
+func GetEnvBool(key string, defaultValue bool) bool {
+	envVal := os.Getenv(key)
+	if envVal == "" {
+		return defaultValue
+	}
+	value, err := strconv.ParseBool(envVal)
```

Comment:
Let's convert envVal to lower before passing it in just to support 🧽 👖 case. Dealers choice if we also support `yes`  / `no`.